app/invocation/invocation.tsx

```diff
@@ -264,6 +264,18 @@ export default class InvocationComponent extends React.Component<Props, State> {
               value={this.getBuildLogs()}
               loading={this.areBuildLogsLoading()}
               expanded={activeTab == "log"}
+              fullLogsFetcher={async () => {
```

Comment:
uber nit: since this isn't using `await`, and the value being returned already has type `Promise<string>`, there's no need to annotate with `async`

Reply:
mmm, I see. `tsserver` emitted an "informational" diagnostic when I didn't annotate it, which is why I had, but I'm happy to just ignore it instead.